### ISSUE TITLE
Build a basic GUI (=basic video editing tool) for (Library 3)

### ISSUE THREAD

**Issue by Freeze-FF** [amosproj/amos2023ws03-gui-frame-diff#50]
## User story
1. As a user
2. I want to have a good GUI that looks like a video editing tool
3. So that I can effectively and efficiently work with the software. 

This ticket depends on #49. #56 
## Acceptance criteria

* Implement the plan/design of what the final GUI should look like, created in #56 
* The GUI should be understandable and intuitive 
* The goal is to create a skeleton that already contains the basic functionality of a video editing tool: selecting and loading videos
* One should be able to select the three video files: Video A, Video B and the diff video 
* GUI: a view with three (Video A and Video B and diff video) videos side by side 
* One should be able to change the current videos by selecting another video from the disk 
* It should contain a run/ pause button (but no functionality for the buttons is required yet!)  
* Images should be displayed correctly
* Different formats and resolutions must be possible. e.g. portrait or panorama format, e.g. 1080 x 1080, but also e.g. 400*800, or any other!

## Definition of done (DoD)
* Create feature branch
* Implement the functionality
* Opened pull request and assigned reviewer
* If necessary: required changes are implemented
* Delete feature branch
* If necessary: Component Test has been written
* Code review has been completed and code has been merged

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by AlperK61** [comment]
looks good. is similar to the one which created by me and @fs3itz
